src/univariate/discrete/betabinomial.jl

```diff
@@ -81,8 +81,9 @@ function kurtosis(d::BetaBinomial)
     return (left * right) - 3
 end
 
-function pdf(d::BetaBinomial, k::Int)
+function pdf(d::BetaBinomial{T}, k::Int) where T
     n, α, β = d.n, d.α, d.β
+    0 <= k <= n || return zero(T)
```

Comment:
Oh really? The current order makes sense to me. I think of it like `contains(haystack, needle)` from Base.